### ISSUE TITLE
Having Separate examples in the repo

### ISSUE THREAD

**Issue by art1373** [facebookexperimental/Recoil#11]
I saw a folder in the other packages repositories called examples. Any new concept introduced into recoil can be put in this folder for others to view and get an idea of it in addition to docs.

Here is my implementation basic(ish) based on docs. We can use it as a basic example.
[example link](https://github.com/art1373/recoiljs-basic)

**Comment by davidmccabe** [comment]
Thanks for providing this example! We have a lot more documentation and examples coming.

**Comment by wburningham** [comment]
@davidmccabe I'm personally interested in explore the code of the canvas application that you demoed in your video. Is there any way to share that application before maybe creating/sharing more polished examples?

**Comment by sibelius** [comment]
this is a recoil todo app based on docs https://github.com/sibelius/recoil-todo

**Comment by drarmstr** [comment]
Yeah, I like the idea of including the live example in the repro and website.  We also have a "playground" that demonstrates some concepts that we should polish up and make live as an even simpler example of all of the core and util helpers.

> @davidmccabe I'm personally interested in explore the code of the canvas application that you demoed in your video. Is there any way to share that application before maybe creating/sharing more polished examples?

@csantos42 and I were thinking it would make sense to include that as an "Advanced Tutorial" next to the "Basic Tutorial"
